app/views/podcasts/show.rss.builder

```diff
@@ -24,20 +24,20 @@ xml.rss 'xmlns:atom' => 'http://www.w3.org/2005/Atom',
 
     xml.image do
       xml.url @podcast.feed_image.url
-      xml.title @podcast.title
+      xml.title @feed.title || @podcast.title
       xml.link @podcast.link
       xml.width @podcast.feed_image.width
       xml.height @podcast.feed_image.height
       xml.description @podcast.feed_image.description unless @podcast.feed_image.description.blank?
     end if @podcast.feed_image
 
-    xml.atom :link, href: (@podcast.url || @podcast.published_url), rel: 'self', type: 'application/rss+xml'
+    xml.atom :link, href: (@feed.url || @feed.published_url), rel: 'self', type: 'application/rss+xml'
 
-    unless @podcast.new_feed_url.blank?
-      xml.itunes :'new-feed-url', @podcast.new_feed_url
+    unless @feed.new_feed_url.blank?
+      xml.itunes :'new-feed-url', @feed.new_feed_url
     end
 
-    xml.itunes :block, 'Yes' if @podcast.itunes_block
+    xml.itunes :block, 'Yes' if @podcast.itunes_block || @feed.try(:private)
```

Comment:
If the feed is private, add `itunes:block`

Reply:
Kind of makes me wonder if we should drop the `itunes_block` column someday.  There does seem to be some cleanup to do, once Publish is using the Feeds-API for a bunch of these columns.

Reply:
seems like another "maybe this should be feed specific?" attribute